src/settings/SettingsScreen.js

```diff
@@ -51,22 +52,34 @@ class SettingsScreen extends React.Component {
     this.props.settingsChange('theme', value ? 'night' : 'default');
   };
 
-  handleNotificationChange = (value) => {
-    toggleOfflinePushNotifications(this.props.auth, value);
-    this.props.settingsChange('notifications', value);
+  handleOfflineNotificationChange = async (value) => {
+    await toggleMobilePushSettings({ auth: this.props.auth, offline: value });
```

Comment:
I don't think we need the `await`. This will make the toggle buttons look laggy.